### ISSUE TITLE
Is there a plan to support pagination of the article list?

### ISSUE THREAD

**Issue by IMBlues** [dragonman225/notablog#27]
Now all articles are on the same page, if pagination can be supported, the homepage of blog will look more concise.

**Comment by dragonman225** [comment]
Agree. Do you know any blogs / blog templates doing this well? Would love to learn from some good designs.

**Comment by IMBlues** [comment]
> Agree. Do you know any blogs / blog templates doing this well? Would love to learn from some good designs.

Just from my personal experience, I prefer to display a small number of the latest articles on the homepage and hide more content behind the "more" button, for examples:
- https://medium.com/
- https://about.instagram.com/blog
- https://lepture.com/

In addition, other themes for reference:
- https://hexo.io/themes/
- http://jekyllthemes.org/
